src/features/filter.ts

```diff
@@ -1,10 +1,56 @@
-export const actions = { /* put action creators here */};
+import { ShowTodos } from '../types/ShowTodos';
 
-const filterReducer = () => {
-  return {
-    query: '',
-    status: 'all',
-  };
+type SetQueryAction = {
+  type: 'filterQuery/SET';
```

Comment:
Create an enum for types to avoid errors connected with typos

Reply:
Refactor